### PR TITLE
Fixed Grammar, Wordiness, and Professionalism

### PR DESCRIPTION
**Pull request by nottisa**
Made cardboard more grammatically correct, less wordy, and more professional!

### CODE REVIEW

File: src/routes/(protected)/settings/your-apps/[id]/+page.svelte [hunk 1]
```diff
@@ -45,7 +45,7 @@
         Editing ➡️ {app.name}
     </h2>
     <h6 class="h6">Note:</h6>
-    <p>Carboard currently only supports upto 512kbs of request body size per app update, this should not harm much functionality but please use compressed images and try splitting the updates id you do encounter any errors.</p>
+    <p>Carboard currently only supports up to 512KB/S of request body size per app update, this should not harm much functionality, however please use compressed images and try splitting the update id you do encounter any errors.</p>
```
Comment (sohamjaiswal) [inline src/routes/(protected)/settings/your-apps/[id]/+page.svelte:48]: kB(s) (plural)

**Review by sohamjaiswal — CHANGES_REQUESTED** [review]
one change, otherwise lgtm 👍

**Comment by nottisa** [comment]
Oh whoops, I forgot it's not kilobytes per second lol

File: src/routes/(protected)/settings/your-apps/[id]/+page.svelte (hunk 1 above)
Comment (nottisa) [reply src/routes/(protected)/settings/your-apps/[id]/+page.svelte]: Fixed with KBs

**Comment by sohamjaiswal** [comment]
lgtm 👍

**Review by sohamjaiswal — APPROVED** [review]

**Pull request merged**